.xo-config.json

```diff
@@ -12,6 +12,14 @@
 	"plugins": ["@vitest"],
 	"nodeVersion": false,
 	"rules": {
+		"no-restricted-syntax": [
+			"error",
+			{
+				"selector":
+					":matches([callee.name=delegate], [callee.name=$], [callee.name=$$], [callee.name=observe], [callee.property.name=querySelector], [callee.property.name=querySelectorAll], [callee.property.name=closest])[arguments.0.value=/,/][arguments.0.value=/.{15,}/]",
```

Comment:
`delegate` is blocked by:

- https://github.com/fregante/delegate-it/issues/35

Also seen in 

- https://github.com/refined-github/refined-github/pull/7689